src/helper.ts

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as gax from 'google-gax';
+import {Descriptors, ClientOptions} from 'google-gax';
+import * as path from 'path';
+import * as gapicConfig from './iam_policy_service_client_config.json';
+import {ProjectIdCallback} from 'google-auth-library';
+import * as protosTypes from '../protos/protos';
+const version = require('../../package.json').version;
+
+export class IamClient {
```

Comment:
Can we get a comment explaining what this thing is?

Reply:
Sure, adding it now, thank you!